beater/config.go

```diff
@@ -23,6 +23,7 @@ type Config struct {
 	Expvar              *ExpvarConfig   `config:"expvar"`
 	Frontend            *FrontendConfig `config:"frontend"`
 	AugmentEnabled      bool            `config:"capture_personal_data"`
+	Agents              []string        `config:"agents"`
```

Comment:
This config is only for metrics, so i suggest we put it under a metrics namespace:
```
metrics:
  agents: ....
```

or see if we can come up with another way to indicate that this config is used to determine which agents we'll be collecting metrics from.

Reply:
i think `agent_metrics` would be an OK name.